examples/mda_useq.py

```diff
@@ -1,7 +1,7 @@
 import useq
 from qtpy.QtWidgets import QApplication
 
-from pymmcore_widgets.useq_widgets import MDASequenceWidget
```

Comment:
Had to do this for circular import but I don't think is a good idea. Need to fix this